gravity-forms/gw-progress-meter.php

```diff
@@ -114,7 +115,8 @@ public function get_count( $atts ) {
 					'where'  => $wpdb->prepare( "
 						WHERE em.form_id = %d
 						AND em.meta_key = %s
-						AND e.status = 'active'\n",
+						AND e.status = 'active'
+						AND e.payment_status = 'Paid'\n",
```

Comment:
@spivurno Should we be adding this conditional in both of these spots? Thinking out loud, what if the form doesn't have payments wired up?

As an aside, should the partial entries bit be moved outside of the else to right above `$sql`?

Reply:
@claygriffiths We'll only want this in the first condition that checks for the presence of the payment_amount. This entry property is only populated if a payment feed is configured (regardless of the presence of product fields on the form).

As for partial entries, I don't think it's possible for a payment to be captured on a partial entry... maybe with Gravity Flow? My gut is to not change that until we have a bug report.

Reply:
> We'll only want this in the first condition that checks for the presence of the payment_amount.

Thought so but wasn't 100% sure 😄

> As for partial entries, I don't think it's possible for a payment to be captured on a partial entry... maybe with Gravity Flow? My gut is to not change that until we have a bug report.

Makes sense!